guidance_pipeline/main.go

```diff
@@ -13,22 +16,84 @@ type Guidance struct {
 	Regs []string `json:"regs"`
 }
 
+var file = flag.String("f", "", "supply a file of URLs to download or a csv file")
+var directory = flag.String("d", "", "give a directory of csv files")
+var url = flag.String("u", "", "give a url to a csv file" )
+var outputDirectory = flag.String("o", "guidance", "give the directory with which you want to output json")
+
 func main() {
-	if len(os.Args) != 2 {
-		fmt.Println("Usage:", "./guidance", "FILE")
+	flag.Usage = func(){
+		fmt.Fprintf(os.Stderr, "Usage: %s [options] [file]\n", os.Args[0])
+		flag.PrintDefaults()
+	}
+	flag.Parse()
+	if flag.NFlag() == 0 {
```

Comment:
Needs a way to check explicitly for `file`, `directory`, or `url`. This passes if only `-o` is provided.